### ISSUE TITLE
Log folding for selected commands

### ISSUE THREAD

**Issue by wsfulton** [travis-ci/travis-ci#2285]
When viewing the logs, the commands run in the before_install section in .travis.yml are folded by default. This is great. in the script section the commands are not folded, but I'd like some of the commands in this section to fold. In particular a 'make distclean' at the end, which is very boring and very verbose. A non-zero exit code from this command should definitely fail the build, so it doesn't seem suitable for the post scripts sections (after_success, after_failure or after_script). 

In summary, is there a way to fold commands in the log within or after the scripts section and still have the exit code pass/fail the build?

**Comment by PeterDaveHello** [comment]
May I ask if there is any method to force extend, not fold the output in some sections? Thanks.
